lib/drivers/docker/container.js

```diff
@@ -274,51 +237,26 @@ Container.prototype.attachMonitor = function() {
   }
 };
 
-Container.prototype.updatePortMapping = function() {
-  this.debug('Docker::Container.updatePortMapping()');
+Container.prototype._resume = function() {
+  this.debug('Docker::Container._resume()');
```

Comment:
`'strong-pm:docker:container:` is already in the debug message, it doesn't have to be repeated. OK to keep it if you want, though.